source/components/dialog/dialog.ts

```diff
@@ -50,15 +55,18 @@ export class DialogComponent extends FormComponent {
 			submitAndClose: () => this.submitAndClose(),
 			size: this.size,
 		});
+		this.dialogOpenStatus.emit(true);
 	}
 
 	close(): void {
 		this.dialogRoot.closeDialog.next(null);
+		this.dialogOpenStatus.emit(false);
 	}
 
 	dismiss(): void {
 		this.dialogRoot.dismissing = true;
 		this.dialogRoot.closeDialog.next(null);
+		this.dialogOpenStatus.emit(false);
```

Comment:
What if we subscribed to `dialogRoot.closeDialog` instead? I'm thinking it's possible for close events to come from outside of this component. (Such as clicking outside the dialog, which will originate from the dialog outlet component.